wp-content/themes/workingnyc/search.php

```diff
@@ -14,17 +14,14 @@
 // Create query
 $wp_query = new WP_Query(array(
   's' => $term,
-  'post_type' => 'any'
+  'post_type' => array('programs', 'jobs')
 ));
 
 // Redo relevanssi query and get posts in Timber format.
 // This block could be made more efficient.
 $relevanssi_query = relevanssi_do_query($wp_query);
 $wp_query_ids = wp_list_pluck($wp_query->posts, 'ID');
 $posts = Timber::get_posts($wp_query_ids);
-$job_or_program_posts = array_filter($posts, function($p) {
```

Comment:
Because the posts are filtered in the WP_Query object, we no longer need to filter them after getting the search results